style.css

```diff
@@ -405,10 +405,10 @@ li {
   flex-direction: row;
   align-items: center;
   justify-content: flex-end;
-  gap: 8px;
-  width: 203px;
-  height: 24px;
-  left: 0;
+  gap: 8.3px;
+  width: 203.4px;
+  height: 24.4px;
+  left: 0.5px;
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix your margin or padding extra space issues. I saw you have got extra space on `left`, `right` and `down`!

![image](https://user-images.githubusercontent.com/63915024/186452875-9c5d0249-df77-4dbe-8c8f-e40cc69720ce.png)